src/Service/Generate/CreateService.php

```diff
@@ -38,10 +38,11 @@ public function run(
         string $command,
         string $table,
         string $className,
-        ?string $namespace = null,
-        ?string $fields = null,
-        ?string $and = null,
-        ?string $tableComment = null
+        string $namespace = null,
+        string $fields = null,
+        string $and = null,
+        string $tableComment = null,
```

Comment:
Why remove null type hint?

Reply:
Because in `PHP` any argument that defaults to `null` is `nullable`.

Reply:
This reduces readability. By fact is legacy syntax. It is better to explicitly use `?string` or `string|null`.

Reply:
Because seeing that the default value is `null`, is it because it accepts `nullable` or not ?

Reply:
It's work, but it reduces readability of code.

Reply:
This is not recommended in [PHP docs](https://www.php.net/manual/en/language.types.declarations.php#language.types.declarations.composite.union) either:

```
It is also possible to achieve nullable arguments by making null the default value. This is not recommended as if the default value is changed in a child class a type compatibility violation will be raised as the null type will need to be added to the type declaration.
```

Reply:
fixed.